task1/js/app.js

```diff
@@ -0,0 +1,46 @@
+let totalZombies = 0;
+let deadZombies = 0;
+let smallZombies = 0;
+let madZombies = 0;
+let strongZombies = 0;
+
+const totalSelector = document.querySelector('.total-zombies');
```

Comment:
"Selector" is not a valid naming for a DOM element
it would be fine to have a name like "totalZombiesElement", "deadZombiesElement"
so "Element" is a better solution